src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -26,16 +27,25 @@ internal sealed class ManagedProjectSystemPackage : AsyncPackage
 
         protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
+            // Here we initialize our internal IPackageService implementations, both in global and project services scope.
+
+            // Get access to global MEF services.
             IComponentModel componentModel = await this.GetServiceAsync<SComponentModel, IComponentModel>();
 
-            IEnumerable<IPackageService> packageServices = componentModel.GetExtensions<IPackageService>();
+            // Get access to project services scope services.
+            IProjectServiceAccessor projectServiceAccessor = componentModel.GetService<IProjectServiceAccessor>();
+
+            // Find package services in global scope.
+            IEnumerable<IPackageService> globalPackageServices = componentModel.GetExtensions<IPackageService>();
+
+            // Find package services in project service scope.
+            IEnumerable<IPackageService> projectServicesPackageServices = projectServiceAccessor.GetProjectService().Services.ExportProvider.GetExportedValues<IPackageService>(ExportContractNames.Scopes.ProjectService);
```

Comment:
nit: `projectServiceScopedPackageServices`

Reply:
The name matches `globalPackageServices` so I'm inclined to keep it. Agree it's a bit weird to have "service" twice in the name. It's local to this method and documented, so hopefully clear enough.